checkov/kubernetes/graph_builder/local_graph.py

```diff
@@ -153,10 +153,10 @@ def _extract_nested_resources_recursive(conf: Dict[str, Any], all_resources: Lis
                 all_resources.append(conf)
                 return
             template[PARENT_RESOURCE_KEY_NAME] = metadata.get('name', "")
+            spec.pop('template', None)
```

Comment:
could you add a test for that case?

Reply:
my last commit aligned the logic with the current tests